roles/varnish/vars/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+varnish:
+  user: varnishlog
+  group: varnish
+  port: "{{ varnish_port|default(80) }}"
+  malloc: "{{ varnish_malloc|default('256m') }}"
```

Comment:
The current variable that is out there in environments is `varnish_memory_allocation`.